aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -875,18 +883,40 @@ std::tuple<at::Tensor, at::Tensor> _efficient_attention_forward(
 
   at::Tensor res;
   at::Tensor logsumexp;
+  at::Tensor seed_t, offset_t;
 
   const bool use_dropout = std::fpclassify(dropout_p) != FP_ZERO;
-  at::PhiloxCudaState rng_engine_inputs;
+  at::PhiloxCudaState philox_state;
```

Comment:
nit: I don't like renames for the sake of renames in large PRs, because I worry that I'll miss an important change as a reviewer and this PR already has a lot of changes.